test/test_direction.py

```diff
@@ -1,47 +1,49 @@
-import unittest
+"""Test direction."""
+import pytest
 from metar.Datatypes import direction
 
-def suite():
-    return unittest.makeSuite(DirectionTest)
 
-class DirectionTest(unittest.TestCase):
```

Comment:
This might be a bit premature, but you might want to use the `parametrize` mark:

```python
@pytest.mark.parametrize('in_value', ["90", 90, 90.0, "E"])
def test_usage_value(dir_value):
    assert direction(dir_value).value() == 90.0
    assert direction(dir_value).string() == "90 degrees"
    assert direction(dir_value).compass() == "E"

@pytest.mark.parametrize('bad_dir', ["North", -10, "361"])
def test_error_checking(bad_dir):
    with pytest.raises(ValueError):
        direction(bad_dir)
```

Reply:
thanks, my current goal is to make Travis-CI happy.  I think such additions could be saved for future PRs as the code base churns.